dask_cuda/proxy_object.py

```diff
@@ -436,6 +436,15 @@ def __setattr__(self, name, val):
             else:
                 object.__setattr__(self._obj_pxy_deserialize(), name, val)
 
+    def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
+        inputs = tuple(
+            o._obj_pxy_deserialize() if isinstance(o, ProxyObject) else o
```

Comment:
Would it also make sense to call `o._obj_pxy_deserialize()` on each value in `kwargs`?

Reply:
Right, handling `kwargs` too now.